utils/sispmctl/Makefile

```diff
@@ -49,9 +54,15 @@ CONFIGURE_ARGS += \
 	--enable-webless \
 	--disable-dependency-tracking
 
+define Package/libsispmctl/install
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/lib/lib$(PKG_NAME)* $(1)/usr/lib/
+endef
+
 define Package/sispmctl/install
 	$(INSTALL_DIR) $(1)/usr/bin
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/$(PKG_NAME) $(1)/usr/bin/
```

Comment:
Please just use the actual name here.  No good with the variable ref

Reply:
Done